tests/unit/lms/views/api/blackboard/sync_test.py

```diff
@@ -1,7 +1,70 @@
+import pytest
+
+from lms.models import Grouping
 from lms.views.api.blackboard.sync import Sync
+from tests.conftest import TEST_SETTINGS
+
+pytestmark = pytest.mark.usefixtures(
+    "assignment_service",
+    "lti_h_service",
+    "grouping_service",
+    "course_service",
+    "blackboard_api_client",
+)
 
 
-def test_it(pyramid_request):
+def test_it(pyramid_request, assert_sync_and_return_groups, blackboard_api_client):
```

Comment:
Only one test as there's no really conditionals or error handling in the view for this version.